src/constants/email-template-ids.ts

```diff
@@ -0,0 +1,27 @@
+// Template ID Constants (template_id values in camdecmpsaux.email_template)
+
+export const EMAIL_TEMPLATE_IDS = {
+  SUBMISSION_CONFIRMATION: 200,
+  SUBMISSION_FEEDBACK: 201,
+  SUBMISSION_FAILURE_USER: 202,
+  SUBMISSION_FAILURE_SUPPORT: 203,
+  QUEUEING_FAILURE_USER: 204,
+  QUEUEING_FAILURE_SUPPORT: 205,
+  MASS_EVALUATION: 206,
+  MATS_SUBMISSION: 207,
+} as const;
+
+// Type for template ID validation
+export type EmailTemplateId = typeof EMAIL_TEMPLATE_IDS[keyof typeof EMAIL_TEMPLATE_IDS];
+
+// Handlebars partials configuration for different template types
+export const EMAIL_TEMPLATE_PARTIALS = {
+  SUBMISSION_FEEDBACK: {
+    basePath: 'email/submissions/feedback/partials',
```

Comment:
This needs to be updated, too:
```suggestion
    basePath: 'templates/email/submissions/feedback/partials',
```

Reply:
Updated 👍